### DISCUSSION TITLE
How to make candles look like this

### DISCUSSION THREAD

**Discussion by jimbow973** [highfestiva/finplot#526] [Q&A]
Hello,

I can't figure out how to make the candles look like this, so that the body width is zero but the open and close prices marks are still visible.

Thanks

![ohlc plot](https://github.com/highfestiva/finplot/assets/118762948/043bcc51-94cf-4943-ac27-5f3fc331199e)

**Comment by highfestiva** [comment]
It's not built into finplot, so you'd have to implement your own renderer. If you do, please add a pull request and I'll integrate it to the current solution.
